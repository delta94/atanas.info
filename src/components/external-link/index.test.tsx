import * as React from 'react';
import { shallow } from 'enzyme';

import { ExternalLink } from '.';

describe('ExternalLink component', () => {
	it('Should render the ExternalLink component', () => {
		const wrapper = shallow(<ExternalLink href="https://google.com">Link</ExternalLink>);

		expect(wrapper).toMatchSnapshot();
	});

	it('Should render the ExternalLink component with more attributes', () => {
		const wrapper = shallow(
			<ExternalLink href="https://google.com" className="link">
				Link
			</ExternalLink>
		);

		expect(wrapper).toMatchSnapshot();
	});
});
