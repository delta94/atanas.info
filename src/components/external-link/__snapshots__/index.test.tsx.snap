// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExternalLink component Should render the ExternalLink component 1`] = `
<a
  href="https://google.com"
  rel="noopener noreferrer"
  target="_blank"
>
  Link
</a>
`;

exports[`ExternalLink component Should render the ExternalLink component with more attributes 1`] = `
<a
  className="link"
  href="https://google.com"
  rel="noopener noreferrer"
  target="_blank"
>
  Link
</a>
`;
